portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.data;
+/**
+ * Comment class that represents the Comment entity stored in datastore
+ *
+*/
+public class Comment {
+  private final String NAME;
+  private final String COMMENT_CONTENT;
+
+  public Comment(String name, String comment) {
+    this.NAME = name;
+    this.COMMENT_CONTENT = comment;
```

Comment:
nit: these should actually be camelCase still (`name` and `content`) -- upper-case is generally only used for `static final` fields

Reply:
Got it!